unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Scene/Main.cs

```diff
@@ -22,6 +22,8 @@ public class Main : MonoBehaviour
         private PerformanceMetricsController performanceMetricsController;
         private EntryPoint_World worldEntryPoint;
 
+        private CrashPayloadPositionTracker positionTracker;
```

Comment:
This private variable is not being used here in the `Main.cs` class. I guess we could remove it, right?